nucleus/constants.py

```diff
@@ -45,6 +45,7 @@
 DATASET_NAME_KEY = "name"
 DATASET_PRIVACY_MODE_KEY = "use_privacy_mode"
 DATASET_SLICES_KEY = "slice_ids"
+DATASET_USE_PRIVACY_MODE = "use_privacy_mode"
```

Comment:
I'm seeing double 🧑‍🦲 🧑‍🦲 . Four `"use_privacy_mode"` constants!

Reply:
oops good catch

Reply:
Where's my laugh? I make good joke! 😄

Reply:
Here you go!